x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/new_features_tour.tsx

```diff
@@ -1,89 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import { noop } from 'lodash';
-import type { FC } from 'react';
-import React, { useEffect, useMemo, useState } from 'react';
-
-import type { EuiTourState } from '@elastic/eui';
-import { EuiSpacer, EuiTourStep, useEuiTour } from '@elastic/eui';
-
-import { NEW_FEATURES_TOUR_STORAGE_KEYS } from '../../../../../common/constants';
-import { useKibana } from '../../../../common/lib/kibana';
-
-import * as i18n from './translations';
-
-const TOUR_STORAGE_KEY = NEW_FEATURES_TOUR_STORAGE_KEYS.RULE_CREATION_PAGE_DEFINE_STEP;
-const TOUR_POPOVER_WIDTH = 300;
-
-const tourConfig: EuiTourState = {
-  currentTourStep: 1,
-  isTourActive: true,
-  tourPopoverWidth: TOUR_POPOVER_WIDTH,
-  tourSubtitle: '',
-};
-
-const stepsConfig = [
-  {
-    step: 1,
-    title: i18n.DATA_SOURCE_GUIDE_TITLE,
-    content: (
-      <span>
-        <p>{i18n.DATA_SOURCE_GUIDE_CONTENT}</p>
-        <EuiSpacer />
-      </span>
-    ),
-    anchor: `#dataSourceSelector`,
```

Comment:
Nit: I think we can remove the id from `x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/index.tsx` because it seems it's not used from anywhere else

Reply:
ah nice find thank you.